packages/renderer/src/components/organisms/WalletManagement/WalletManagement.tsx

```diff
@@ -0,0 +1,107 @@
+// =============================================================================
+//  Copyright 2022 Haveno
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// =============================================================================
+
+import { useState } from "react";
+import { useForm } from "@mantine/hooks";
+import { Group, Space, Stack } from "@mantine/core";
+import { Button } from "@atoms/Buttons";
+import { TextInput } from "@atoms/TextInput";
+import { FormattedMessage } from "react-intl";
+import { LangKeys } from "@constants/lang";
+import { showNotification } from "@mantine/notifications";
+import { WalletItem } from "./WalletItem";
+import { useValidatePassword } from "@hooks/haveno/useValidatePassword";
+
+export function WalletManagement() {
+  const [validate, setvalidate] = useState(false);
```

Comment:
This naming is very confusing. Better alternatives: `isRevealed`, `setRevealed`, `isHidden`, `setHidden`, etc